test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Can we have an elaborated name for this file (like perf-test-helper.sh or something)? test.sh seems very generic

Reply:
I changed the name to `perf-test.sh`. My initial plan was to use a simple to remember and short name, which is easy to use in the console. After renaming it, I am using an alias for the same purpose.